clang/include/clang/Sema/ParsedAttr.h

```diff
@@ -615,7 +609,12 @@ class ParsedAttr final
   bool diagnoseAppertainsTo(class Sema &S, const Decl *D) const;
   bool diagnoseAppertainsTo(class Sema &S, const Stmt *St) const;
   bool diagnoseMutualExclusion(class Sema &S, const Decl *D) const;
-  bool diagnoseMutualExclusion(class Sema &S, const Stmt *St) const;
+  // This function stub exists for parity with the declaration checking code so
```

Comment:
I do not like this design but I could not think of a more elegant solution. If you have better ideas that don't result in code duplication in [`checkCommonAttributeFeatures()`](https://github.com/intel/llvm/blob/sycl/clang/lib/Sema/SemaAttr.cpp#L1193), I'm all ears.